tensorflow/python/tools/saved_model_cli.py

```diff
@@ -184,7 +184,7 @@
     ' object.')
 
 _SMCLI_TARGET_TRIPLE = flags.DEFINE_string(
-    name='target_triple', default='x86_64-pc-linux',
+    name='target_triple', default='',
     help='Triple identifying a target variation, containing information such as'
     ' processor architecture, vendor, operating system, and environment. '
     'Defaults to \'x86_64-pc-linux\'.')
```

Comment:
Update help string to reflect new default value.

Reply:
Thank you. Should be fixed now. Sorry about that.

Reply:
All good. Thanks for the change!